### DISCUSSION TITLE
Automatically create an issue in the issue tracker. (XL)

### DISCUSSION THREAD

**Discussion by Sebastiaan127001** [ICTU/quality-time#3993] [Ideas]
As a Quality Time user 
I want to create a ticket in the issue tracker
So that I can save time

Related issues:
- #2931

**Comment by fniessink** [comment]
It is possible to have multiple projects for one repository, so I opened a ["Epics" board](https://github.com/ICTU/quality-time/projects/3) and created an issue (epic) for this discussion there. Please add other epics on the Epics board.
